Dockerfile

```
@@ -93,8 +93,8 @@ RUN rm -vf /charts/*.sh /charts/*.md
 # This image includes any host level programs that we might need. All binaries
 # must be placed in bin/ of the file image and subdirectories of bin/ will be flattened during installation.
 # This means bin/foo/bar will become bin/bar when rke2 installs this to the host
-FROM rancher/k3s:v1.20.11-k3s2 AS k3s
-FROM rancher/hardened-kubernetes:v1.20.12-rke2r1-build20211028 AS kubernetes
+FROM rancher/k3s:v1.20.13-k3s1 AS k3s
+FROM rancher/hardened-kubernetes:v1.20.13-rke2r1-build20211119 AS kubernetes
 FROM rancher/hardened-containerd:v1.4.11-k3s1-build20211004 AS containerd
 FROM rancher/hardened-crictl:v1.19.0-build20210223 AS crictl
 FROM rancher/hardened-runc:v1.0.2-build20210915 AS runc
```

Updated builds of these that have been rebuilt with the new golang version are also available and should be used for this release. Same with the etcd image (`v3.4.13-k3s3-build20211118`)
```suggestion
FROM rancher/hardened-containerd:v1.4.12-k3s1-build20211118 AS containerd
FROM rancher/hardened-crictl:v1.20.0-build20211118 AS crictl
FROM rancher/hardened-runc:v1.0.2-build20211118 AS runc
```